csvparser.go

```diff
@@ -271,3 +272,56 @@ func (c *CsvParser[ReadTo]) runOnFinish() {
 		c.onFinish()
 	}
 }
+
+/*
+Given array of structs from the parsed csv, prints out the contents of the file parsing in a nice string format.
+The input of the function can take in any type of structs with any number of fields within the struct.
+*/
+func ParseToString(arr []interface{}) string {
```

Comment:
Thank you for contributing!
I like the idea of having a ParseToString helper. Maybe we could just put this function in a new .go file? Maybe a file with the name `helpers.go`.
Also, could you add a unit test or two? It would be nice to have everything tested.